### PR TITLE
Ocp cos ra 5

### PR DESCRIPTION
**Pull request by thamill**
Vulnerability scanner policy and procedure is managed by the organization - neither COS nor OCP is a vulnerability scanning appliance.

### CODE REVIEW

File: coreos-4/policies/RA-Risk_Assessment/component.yaml [hunk 1]
```diff
@@ -117,34 +117,33 @@
...
+        Organizational procedures to initially scan for vulnerabilities and to perform a scan
+        when new vulnerabilities are identified and reported are outside the scope of RHCOS
+        configuration.
     - key: b
       text: |
-        A control response is planned. Progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-352
+        Organizational procedures to facilitate interoperability and vulnerability management
+        process automation are outside the scope of RHCOS configuration.
```
Comment (shawndwells) [inline coreos-4/policies/RA-Risk_Assessment/component.yaml]: NACK. For RA-5(b) a response that outline OpenShift's embedded vulnerability scanning capabilities would be appropriate.

File: coreos-4/policies/RA-Risk_Assessment/component.yaml [hunk 2]
```diff
@@ -117,34 +117,33 @@
...
-        applicable to the configuration of CoreOS.
+        Organizational procedures to analyze vulnerability scan outputs from security controls
+        are outside the scope of RHCOS configuration.
     - key: d
       text: |
-        A control response is planned. Progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-352
+        Organizational procedures to remediate legitimate vulnerabilties within organization
+        determined response times is outside the scope of RHCOS configuration.
```
Comment (shawndwells) [inline coreos-4/policies/RA-Risk_Assessment/component.yaml]: For RA-5(d) a response detailing how to remediate vulnerabilities natively in OpenShift would be appropriate. For example Clair/Quay at the registry level or OpenSCAP during CI/CD.

Comment (thamill) [reply coreos-4/policies/RA-Risk_Assessment/component.yaml]: I added language supporting Clair/Atomic remediation.
The Clair and Atomic scanners can be hosted in OpenShift, but could be replaced by other products within an organization's vulnerability management program.

**Comment by redhatrises** [comment]
@thamill need the build commits removed

File: coreos-4/policies/RA-Risk_Assessment/component.yaml [hunk 3]
```diff
@@ -117,34 +117,33 @@
...
-  implementation_status: partial
+  implementation_status: not applicable
   narrative:
     - key: a
       text: |
-        This control reflects organizational procedures/policies, and is not
-        applicable to the configuration of CoreOS.
+        Organizational procedures to initially scan for vulnerabilities and to perform a scan
+        when new vulnerabilities are identified and reported are outside the scope of RHCOS
+        configuration.
```
Comment (redhatrises) [inline coreos-4/policies/RA-Risk_Assessment/component.yaml:126]: Is there any documentation for this?

**Comment by redhatrises** [comment]
The comments in this PR haven't been addressed or resolved in a few months. This PR looks frozen. Closing. Please re-open this PR when the comments are addressed appropriately.

**Pull request closed without merging**